corehq/apps/export/templates/export/partial/daily_export_move_notifier.html

```diff
@@ -21,10 +21,3 @@ <h4 class="modal-title">
         </div>
     </div>
 </div>
-{% addtoblock js-inline %}
-<script>
-    $(window).on('load', function(){
-        $('#dailySavedExportNotifier').modal('show');
```

Comment:
this just gets handled by magic bootstrap js?

Reply:
Kind of - modals actually default to being open. Typically we add the `fade` class, which hides them, so then you need js to open them. So removing `fade` removed the need for js.